kci_test

```diff
@@ -77,21 +77,31 @@ class cmd_validate(Command):
 
 class cmd_list_jobs(Command):
     help = "List all the jobs that need to be run for a given build and lab"
-    args = [Args.bmeta_json, Args.lab_config]
-    opt_args = [Args.dtbs_json, Args.user, Args.lab_token, Args.lab_json]
+    args = [Args.lab_config]
+    opt_args = [Args.user, Args.lab_token, Args.lab_json,
+                Args.build_output, Args.install_path]
 
     def __call__(self, test_configs, lab_configs, args):
-        bmeta, dtbs = kernelci.build.load_json(args.bmeta_json, args.dtbs_json)
+        path_args = (args.build_output, args.install_path)
+        if not any(path_args) or all(path_args):
+            print("Either --build-output or --install-path is required")
+            return False
+
+        install = (
+            args.install_path
+            or kernelci.build.Step.get_install_path(None, args.build_output)
```

Comment:
I like how or is concise, but I guess that more pythonic way would be to use 
`args.install_path if args.install_path else kernelci.build.Step.get_install_path(None, args.build_output)`

Reply:
Fixed